src/GraphQLController.php

```diff
@@ -86,6 +98,61 @@ protected function queryContext(string $query, ?array $params, string $schema)
         }
     }
 
+    /**
+     * Note: it's expected this is called even when APQ is disabled to adhere
+     *       to the negotiation protocol.
+     *
+     * @param  string  $schemaName
+     * @param  array<string,mixed>  $input
+     * @return string
+     */
+    protected function handleAutomaticPersistQueries(string $schemaName, array $input): string
+    {
+        $query = $input['query'] ?? '';
+        $apqEnabled = config('graphql.apq.enable', false);
+
+        // Even if APQ is disabled, we keep this logic for the negotiation protocol
+        $persistedQuery = $input['extensions']['persistedQuery'] ?? null;
```

Comment:
Removed `data_get` and applied null coalesce; also somewhere below too

Reply:
Ah! I didn't know he could handle nested array it like this! Cool, thanks!